README.md

```diff
@@ -4,17 +4,19 @@ Running project: [VIEW](https://medid.herokuapp.com/)
 
```

Comment:
You can now use coveralls https://coveralls.io/ to see code coverage percentage.